### PR TITLE
Remove mentions of rancher telemetry

### PR DESCRIPTION
**Pull request by adamkpickering**
### Summary

Fixes: #12639

This PR removes code related to telemetry. This removal is planned for 2.11. Related PRs:

- https://github.com/rancher/rancher/pull/48031
- https://github.com/rancher/rancher-docs/pull/1567

The JIRA issue for this is SURE-9295.

I am not super familiar with this repository so if there is anything that needs changing please let me know. But I think this should be fairly straightforward.

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Comment by richard-cox** [comment]
Note for whoever picks this up, we should review this from a UX and automated test perspective as well as functional and code.

@rancher-max Should there also be UI QA on the changes?

@adamkpickering Feel free to reference jira as SURE-x

**Comment by nwmac** [comment]
@adamkpickering So, there will be no telemetry in 2.11, is that right?

I think the only thing we have in the UI is the checkbox on the setup screen to allow users to opt-out of telemetry - is this what we need to remove from the UI?

**Comment by adamkpickering** [comment]
> @adamkpickering So, there will be no telemetry in 2.11, is that right?

Yes that is right.

> I think the only thing we have in the UI is the checkbox on the setup screen to allow users to opt-out of telemetry - is this what we need to remove from the UI?

I believe so. The only use of the telemetry we are getting rid of was in multi-cluster apps (`pkg/multiclustermanager` in `rancher/rancher`), which are now deprecated. That being said, I'm not super familiar with `rancher/dashboard`, so I may have missed something.

**Comment by adamkpickering** [comment]
@richard-cox @nwmac is there any chance you could give this a look? It would be nice to get it merged so that it doesn't go stale.

File: shell/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -2940,7 +2940,6 @@ istio:
   policy: Enable Policy
   pilotRequired: Pilot must be enabled in order to enable Kiali
   poweredBy: Powered by <a target="_blank" rel="noopener noreferrer nofollow" href='https://istio.io/latest/'>Istio</a>
-  telemetry: Enable Telemetry
```
Comment (richard-cox) [inline shell/assets/translations/en-us.yaml:2943]: This is in the istio block, looking at the code it relates to the istio chart and a telemetry setting there. Are you sure it relates to rancher telemetry?

**Review by richard-cox — CHANGES_REQUESTED** [review]
One comment regarding translations, and i've pushed one tweak.

The removal of content made a bad vertical alignment situation much worst  and i've added a short term fix (see before / after below). Looks like we have a min-height block below to show errors (this avoids bobbling content when the errors show) however it makes the vertical alignment weird when there are no errors. 

Before

image

After

image

File: shell/assets/translations/en-us.yaml (hunk 1 above)
Comment (adamkpickering) [reply shell/assets/translations/en-us.yaml:2943]: Good catch, I shouldn't have removed this. Fixing it now, as well as it's counterpart in shell/assets/translations/zh-hans.yaml.

**Comment by adamkpickering** [comment]
Thanks for the review and tweak @richard-cox  :smile: 

It should be good for another look now.

**Pull request merged**